### PR TITLE
Wdfn 263 address aria violations

### PR DESCRIPTION
**Pull request by ayan-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by mbucknell** [comment]
I'm inclined to leave the h3 tag in the provisional alert as is. This comes from the USWDS standards. We could potentially use the h2 tag and then style appropriately. Did you test to see what it looks like if you just use a standard <div>?

**Comment by danielnaab** [comment]
It's possible to do `@include h3()` in sass. Curious why that's a violation?

**Comment by ayan-usgs** [comment]
> I'm inclined to leave the h3 tag in the provisional alert as is. This comes from the USWDS standards. We could potentially use the h2 tag and then style appropriately. Did you test to see what it looks like if you just use a standard div tag ?

I'm inclined to leave it as it is as well. It's definitely a bit more subdued as a div with bolded text.

**Comment by ayan-usgs** [comment]
> It's possible to do @include h3() in sass. Curious why that's a violation?

I think it's because we have an h1 tag for the station name... then the next header is an h3 without there having been an h2.

File: assets/src/scripts/components/hydrograph/index.js [hunk 1]
```diff
@@ -236,6 +236,7 @@ const createTitle = function(elem) {
 const watermark = function (elem) {
     elem.append('img')
         .classed('watermark', true)
+        .attr('alt', 'USGS logo')
```
Comment (danielnaab) [inline assets/src/scripts/components/hydrograph/index.js]: This is a nit, but I think this should represent what's on the logo.. perhaps: "USGS - science for a changing world".

Comment (ayan-usgs) [reply assets/src/scripts/components/hydrograph/index.js]: I'm cool with that.

**Pull request merged**